posts/inside-rust/2023-12-22-trait-system-refactor-initiative.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Rustc Trait System Refactor Initiative Update: A call for testing"
+author: lcnr
+team: The Rustc Trait System Refactor Initiative <https://github.com/rust-lang/trait-system-refactor-initiative/>
+---
+
+It has been nearly half a year since [our last update][prev]. See that update for a description of our work. We have continued to make big progress on the new solver since then, mostly focusing on getting the solver ready for use in coherence. We changed the unstable compiler flag to enable the new solver: you can now use `-Znext-solver=globally` to enable it everywhere and `-Znext-solver=coherence` to enable the new solver only for coherence checking.
```

Comment:
Sorry, small nit: I would strongly recommend providing a summary of the work that y'all have been doing here instead of linking people to the previous post.

Reply:
agreed. added a short summary of our work to the start of the post.

not 100% happy with the wording but good enough imo